res/css/structures/_HomePage.scss

```diff
@@ -56,14 +56,14 @@ limitations under the License.
     }
 
     .mx_HomePage_default_buttons {
+        display: flex;
         margin: 60px auto 0;
         width: fit-content;
 
         .mx_AccessibleButton {
             padding: 73px 8px 15px; // top: 20px top padding + 40px icon + 13px margin
 
             width: 160px;
-            height: 132px;
```

Comment:
Rather than removing the height I would suggest changing this property to `min-height`.

This way we ensure a consistent vertical rythm even on languages that might be shorter than french or english

Reply:
I just changed `height` to `min-height` on my latest commit. Thank you so much for your feedback! 🎉